libsolidity/analysis/ImmutableValidator.cpp

```diff
@@ -29,7 +29,7 @@ void ImmutableValidator::analyze()
 {
 	m_inConstructionContext = true;
 
-	auto linearizedContracts = m_currentContract.annotation().linearizedBaseContracts | boost::adaptors::reversed;
+	auto linearizedContracts = ranges::views::reverse(m_currentContract.annotation().linearizedBaseContracts);
```

Comment:
Is there a problem with chaining it using `|`?
```suggestion
	auto linearizedContracts = m_currentContract.annotation().linearizedBaseContracts | ranges::views::reverse;
```

Reply:
I think the PR tried to change it to a consistent function call case. If we are goal is to use the pipe notation consistently then it should be changed. However it was inconsistent prior to this PR already.

Reply:
No problem, I can use | syntax everywhere

Reply:
Done